### PR TITLE
Add config for babel v7

### PR DESCRIPTION
**Pull request by adam-h**

### CODE REVIEW

**Comment by adam-h** [comment]
@mdiebolt I've started converting mobile to babel v7 (to use the same fancy async loading you put into viz).

This should work as a shared config for all our repos. Will this work for you in viz?

To install use the following in package.json:

    "@polleverywhere/babel-preset-pe": "1.1.1-babel-7.0",

**Comment by mdiebolt** [comment]
Yeah, this looks good. We probably won't switch Viz over until text well and ticker updates go out, since we've spent a lot of time on infrastructure lately.

**Comment by adam-h** [comment]
This is now live in mobile, is anyone (@mdiebolt @nguyenj @steel @dannobytes) against me merging this as a v2 of the babel config? Can continue 1.x in a branch if needed

**Comment by nguyenj** [comment]
@adam-h I updated this a bit, the presets now includes current [specs](https://github.com/babel/babel/blob/90566103a6bf3cc84bb043fd5342fb588769f16c/packages/babel-preset-env/package.json#L14-L49) and [stage 3 specs](https://github.com/babel/babel/blob/90566103a6bf3cc84bb043fd5342fb588769f16c/packages/babel-preset-stage-3/package.json#L11-L19).

**Comment by adam-h** [comment]
@nguyenj Updated this to the latest beta version, and have a branch using this in mobile. Once I test it a little more I'll merge this to master and the same for mobile. Then we should perhaps work to get viz and web presenter on the same.

File: babel-preset-pe/package.json [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
   "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "7.0.0-beta.44"
+  },
   "dependencies": {
```
Comment (nguyenj) [inline babel-preset-pe/package.json]: @adam-h Not sure if this was intentional or not; but these a la carte plugins here are not necessary from my previous commit. The latest `preset-env` and `preset-stage-3` will cover our most if not all of what we need. We will still need the runtime and polyfill though.

Comment (nguyenj) [reply babel-preset-pe/package.json]: For reference: https://github.com/polleverywhere/js-config/pull/14/commits/fa99eaf46bee75b3688616523507dab6511d0756#diff-a1117a5933c1fe0b9b78483078f534ccR16

Comment (adam-h) [reply babel-preset-pe/package.json]: @nguyenj That's been deprecated as of beta52

>Deprecating the yearly/stage presets in v7 (will remove next beta). Ran npm deprecate on @babel/preset-es2015, @babel/preset-es2016, @babel/preset-es2017, @babel/preset-stage-0, @babel/preset-stage-1, @babel/preset-stage-2, @babel/preset-stage-3 only for versions >v7.0.0-beta.52

Comment (nguyenj) [reply babel-preset-pe/package.json]: I don't follow, I'm looking at beta52 commit and all the plugins are listed.

- https://github.com/babel/babel/blob/663c96db0417d2e9283ebd0c378f2611f49448c4/packages/babel-preset-env/package.json
- https://github.com/babel/babel/blob/663c96db0417d2e9283ebd0c378f2611f49448c4/packages/babel-preset-stage-3/package.json

Comment (adam-h) [reply babel-preset-pe/package.json]: It's written in the Summary [here](https://github.com/babel/babel/releases/tag/v7.0.0-beta.52) for the main release

Comment (nguyenj) [reply babel-preset-pe/package.json]: Ah thanks!

**Comment by adam-h** [comment]
@nguyenj How is this for merging as v2? I've got this working in a mobile [PR](https://github.com/polleverywhere/mobile/pull/324) and if you confirm viz is good with it too we should be set :)

Note there's a hack for shopify draggable in the mobile webpack config, as they have issues with babel-7 at the moment.

**Comment by nguyenj** [comment]
@adam-h I assume this is the correct version `2.0.0-babel-7.3` to use?

**Comment by nguyenj** [comment]
@adam-h I added `@polleverywhere/babel-preset-pe@2.0.0-babel-7.3` to viz.

**Comment by adam-h** [comment]
Thanks John for the second set of eyes checking it :)

**Comment by nguyenj** [comment]
Yeah thanks for getting this in!

**Pull request merged**